### ISSUE TITLE
Accessibility - WCAG 2.4.7 Focus visible, skip links not visible

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#146]
There are no skip links - so they can’t be visible. See picture, they appear when tabbing from the top.

Good example: www.admin.ch/
![image](https://cloud.githubusercontent.com/assets/7510842/3227048/93fd5e5a-f076-11e3-8d3a-b604e3818d4e.png)
